treeherder/client/thclient/client.py

```diff
@@ -674,16 +674,16 @@ def get_machine_platforms(self):
         """
         return self._get_json(self.MACHINE_PLATFORM_ENDPOINT)
 
-    def get_resultsets(self, project, **params):
+    def get_pushes(self, project, **params):
```

Comment:
Should I add a test for `get_resultsets()` [here](https://github.com/maxchehab/treeherder/blob/master/tests/client/test_treeherder_client.py#L270)?